README.md

```diff
@@ -0,0 +1,17 @@
+# sywtltc_chapter_3_4
+Code challenge for So you want to learn to code 3.4
+
+Inherit some very interesting code from a former employee and reworked it to get it running and readable.
+
+purchase_tickets contains the subfunctions charge, dispense_ticket, and remove_seat
+refund_tickets contains add_funds, remove_ticket, and add_seat.
+
+It is only necessary to use purchase_tickets and refund_tickets. The subfunctions should not be necessary on their own.
```

Comment:
Break things into two files - purchase_tickets and refund_tickets. 

You can create a third file that has your shared globals - so purchase_tickets and refund_tickets will both import something like movie_database.py that has the globals like your price in them. 

All functions that you don't want to be used on their own, prepend them with a _, so add_funds, since it shouldn't be used on its own, should be renamed _add_funds

Python doesn't do this well, but we're getting closer and closer to 'object orientation' and what you need for add_funds, etc... that aren't necessary on their own is called "encapsulation". Other languages have sections called 'public' and 'private' for this.

We'll simulate that here.